src/display/api.js

```diff
@@ -245,6 +246,7 @@ function getDocument(src, pdfDataRangeTransport,
   }
 
   params.rangeChunkSize = params.rangeChunkSize || DEFAULT_RANGE_CHUNK_SIZE;
+  var canvasFactory = params.canvasFactory || new DOMCanvasFactory();
```

Comment:
Based on https://github.com/mozilla/pdf.js/pull/8002#discussion_r99022111, this line should be removed.